TopMovies/View/CategoryVC/MoviesCategoryVC.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  MoviesCategoryVC.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 17.02.2021.
+//
+
+import UIKit
+import SnapKit
+
+// MARK: - Props struct
+struct MoviesCategoryVCProps {
+  let categoryId: MovieCategory.ID
+  let categoryName: String
+  let loadProgress: LoadProgress
+  var moviesState: MoviesState
+  
+  struct LoadProgress {
+    let isReloadInProgress: Bool
+    let isLoadMoreInProgress: Bool
+  }
+  struct MoviesState {
+    let hasNewData: Bool
+    let moviesUsage: RequestType
+    var movies: [MovieTableViewCellProps]
+  }
```

Comment:
And I don't really think that you need those types since nothing will be changed if you just add those properties in Props struct.

Reply:
It's not even nullable, cz you expect default values.